tools/ci/run_tc.py

```diff
@@ -161,15 +163,44 @@ def install_webkitgtk_from_apt_repository(channel):
     run(["sudo", "apt-get", "-qqy", "-t", "bionic-wpt-webkit-updates", "install", "webkit2gtk-driver"])
 
 
+# Download an URL in chunks and saves it to a file descriptor (truncating it)
+# It doesn't close the descriptor, but flushes it on success.
+# It retries the download in case of ECONNRESET up to max_retries.
+def download_url_to_descriptor(fd, url, max_retries=3):
+    download_succeed = False
+    if max_retries < 0:
+        max_retries = 0
+    current_retry = 0
+    while True:
```

Comment:
I'd prefer the `while` loop itself examined the retry count rather than having a separate condition further down the loop. It could just be `for retry in range(max_retries + 1)` or similar.

Reply:
Ok. Pushed a new commit with that change